chaoscenter/authentication/go.mod

```diff
@@ -5,48 +5,74 @@ go 1.20
 require (
 	github.com/coreos/go-oidc/v3 v3.1.0
 	github.com/gin-contrib/cors v1.3.1
-	github.com/gin-gonic/gin v1.7.7
+	github.com/gin-gonic/gin v1.9.0
```

Comment:
Since we have upgraded the version of gin gonic, can you confirm if it is working fine and there are no breaking changes?

Reply:
Checking it...